### ISSUE TITLE
Task Scheme incorrect error on multi-line jinja2

### ISSUE THREAD

**Issue by nlvw** [ansible/schemas#413]
The yaml below works correctly when run but this schema, used with ansible-lint, throws an error.

<img width="453" alt="image" src="https://user-images.githubusercontent.com/5723368/200680353-f2d2bf2d-af32-4633-b53e-83993d652a1c.png">

```yaml
- name: update basic package list
  set_fact:
    pkgs_basic: "{{ pkgs_basic + lookup('ansible.builtin.vars', item) }}"
  with_items: >-
    {{ query(
      'ansible.builtin.varnames',
      '^pkgs_basic_.*$',
      '^pkgs_el' + ansible_distribution_major_version + '$',
      '^pkgs_el' + ansible_distribution_major_version + '_.*$'
    ) }}
```
